src/app/features/settings-dropdown/settings-dropdown.tsx

```diff
@@ -92,7 +93,7 @@ export function SettingsDropdown() {
               data-testid={SettingsSelectors.ToggleTheme}
               onClick={wrappedCloseCallback(() => {
                 void analytics.track('change_theme_menu_item');
-                navigate(RouteUrls.ChangeTheme);
+                navigate(RouteUrls.ChangeTheme, { relative: 'path' });
```

Comment:
It makes sure to add the route to the existing url, rather than replacing it. Without it'll try going `/unlock` → `/sign-out`, rather than`/unlock` → `/unlock/sign-out`